staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -200,10 +200,17 @@ type GroupResources struct {
 	// The empty string represents the core API group.
 	// +optional
 	Group string
-	// Resources is a list of resources within the API group.
-	// Any empty list implies every resource kind in the API group.
+	// Resources is a list of resources within the API group. Specific subresources
+	// must be matched using a "/" to indicate the subresource. For example "pods" does
+	// not match "pods/logs".
+	// An empty list implies every resource kind in the API group.
```

Comment:
Do we want to add a wildcard for subresource matching? E.g. `pods/*` matches all pod subresources? I'm not sure whether this would be useful.

Alternatively, is matching all subresources the common case? In that case, we might want to make `pods` match `pods/logs`, and require another rule to match `pods/logs` separately if desired.

WDYT?